pypesto/objective/aggregated.py

```diff
@@ -59,19 +60,29 @@ def __deepcopy__(self, memodict=None):
 
         return other
 
-    def check_mode(self, mode) -> bool:
+    def check_mode(self, mode: str) -> bool:
         return all(
             objective.check_mode(mode)
             for objective in self._objectives
         )
 
-    def check_sensi_orders(self, sensi_orders, mode) -> bool:
+    def check_sensi_orders(
+        self,
+        sensi_orders: Tuple[int, ...],
+        mode: str,
+    ) -> bool:
         return all(
             objective.check_sensi_orders(sensi_orders, mode)
             for objective in self._objectives
         )
 
-    def call_unprocessed(self, x, sensi_orders, mode) -> ResultDict:
+    def call_unprocessed(
+        self,
+        x: np.ndarray,
+        sensi_orders: Tuple[int, ...],
+        mode: str,
+        **kwargs,
+    ) -> ResultDict:
         return aggregate_results([
             objective.call_unprocessed(x, sensi_orders, mode)
```

Comment:
```suggestion
            objective.call_unprocessed(x, sensi_orders, mode, **kwargs)
```
Or remove **kwargs three lines above at is not used in this class otherwise?